met-web/src/components/engagement/listing/index.tsx

```diff
@@ -102,7 +103,16 @@ const EngagementListing = () => {
             disablePadding: false,
             label: 'Status',
             allowSort: true,
-            getValue: (row: Engagement) => row.engagement_status.status_name,
+            getValue: (row: Engagement) => {
+                if (row.engagement_status.status_name === 'Published') {
+                    return (
+                        row.engagement_status.status_name +
+                        '/' +
+                        Object.keys(SubmissionStatus)[Object.values(SubmissionStatus).indexOf(row.submission_status)]
```

Comment:
I think we can simplify this, isn't SubmissionStatus[row.submission_status] enough here?

Reply:
made the suggested changes